pyscript.core/src/stdlib/pyweb/media.py

```diff
@@ -0,0 +1,91 @@
+from pyodide.ffi import to_js
```

Comment:
I think this is the main reason PyWeb code doesn't work on MicroPython ... which is a bit of a bummer.

/cc @ntoll this is one API Damien might offer sooner than later.

Reply:
Exactly. I want to work on making it agnostic to the interpreter today/tomorrow.

Reply:
Yeah, I've opened a discussion after seeing this indeed and proposed possible solution that shouldn't trap your work in here neither ... as in: "*we could, and probably should, normalize needed APIs here and there in PyScript, not use the pyodide specific API to do so*" ... that would make you import `to_js` from `pyscript` and call it a day, other cases might need more discussions behind the scene though.

The discussion: https://github.com/pyscript/pyscript/discussions/1952